atd-vzv/src/views/nav/SideDrawer.js

```diff
@@ -55,12 +55,21 @@ const StyledDrawerHeader = styled.div`
 `;
 
 const StyledDrawer = styled.div`
+  /* Disable side drawer in desktop viewport */
+  #summary-side-drawer {
+    @media only screen and (min-width: ${responsive.bootstrapMedium}px) {
+      display: none;
+    }
+  }
```

Comment:
Clever stuff! I didn't realize you could do that with the `styled.div` method

Reply:
Thanks - removing this side bar turned out to be a little more work than I thought. Hoping that having the CSS within the components makes it easier to see what styling was necessary.